core/src/main/scala/org/scalafmt/config/ScalafmtConfig.scala

```diff
@@ -209,6 +223,7 @@ case class ScalafmtConfig(
     danglingParentheses: Boolean = false,
     poorMansTrailingCommasInConfigStyle: Boolean = false,
     bestEffortInDeeplyNestedCode: Boolean = false,
+    verticalMultiline: Boolean = false,
```

Comment:
It is odd because this is a whole style, but only for `def` sites (as opposed to a "style" like `IntelliJ`). This "mini-style" will also override things like continuation.defnSite (I'll make sure to mention it in the doc).